extension-manifest-v2/src/classes/BugDb.js

```diff
@@ -124,13 +124,17 @@ export class BugDb {
 			}), {});
 
 			oldKnownAppIds.forEach((app_id) => {
-				const category = this.db.apps[app_id].cat;
+				const app = this.db.apps[app_id];
+				if (!app) { return; }
+				const category = app.cat;
 				const vote = selectedApps.hasOwnProperty(app_id) ? 1 : -1;
 				majority[category] = (majority[category] || 0) + vote;
 			});
 
 			this.conf.new_app_ids.forEach((app_id) => {
-				const { cat: category } = this.db.apps[app_id];
+				const app = this.db.apps[app_id];
```

Comment:
Can this happen? I can understand that old ones don't have a entry after an upgrade, but for new ones, it shouldn't happen, right?

Reply:
it happens because https://github.com/ghostery/ghostery-extension/pull/1126